### ISSUE TITLE
Simplify/minisize number of open namespace instructions

### ISSUE THREAD

**Issue by MangelMaxime** [Dzoukr/Feliz.Bulma#63]
This issue originates from https://twitter.com/cognipla/status/1341314506723643392

It should actually be possible to have all the packages opened with a single `open Feliz.Bulma` instruction. We are actually already doing it in Fulma for the extension module.

![image](https://user-images.githubusercontent.com/4760796/102876422-74ec2f80-4445-11eb-9fad-6314745323a8.png)

See how I have access to `Checkradio` and `Calendar` using only `open Fulma.Extensions.Wikiki`.

All the package seems to create their own namespace under `Feliz.Bulma.X`

![image](https://user-images.githubusercontent.com/4760796/102874964-7ae11100-4443-11eb-9c07-f4cff1e5e303.png)

![image](https://user-images.githubusercontent.com/4760796/102875042-8fbda480-4443-11eb-83a1-c54dd2403428.png)

If we change all those declarations to `namespace Feliz.Bulma` then the user should be able to just use `open Feliz.Bulma` and then have access to all the prefixed modules:

```fs
open Feliz.Bulma

Checkradio.checkbox [ ]

Bulma.button.div [ ]

etc. 
```

Things to check for the impact of such a change:
- Namespace pollution only give access to the minimal stuff needed. Using private or internal modules should help here not exposed everything to the world.
- `ElementLiterals` still works as expected because it will be declared several times under the same namespace. They are declared in private so perhaps it doesn't cause problems

**Comment by Dzoukr** [comment]
That sounds really good. One thing I need to consider is how to version such change. From SemVer perspective it's major (breaking) change, because all already used namespaces like `Feliz.Bulma.Checkradio` won't exist anymore. But on the other hand it doesn't bring anything new from functionality point of view. 🤔

**Comment by MangelMaxime** [comment]
@Dzoukr There is nothing wrong in increasing the version number ^^. I know we are always hesitant in increasing major version for things like that but that's still breaking changes.

From a user point of view, I prefer a clear message. Here we are breaking the surfacing API so you have a new major version to show it.

For me I see SemVer like that:
- Increase major version when there is a breaking change, how small it can be
- Increase "medium" (don't know the word^^) version, when adding functionalities or when changing internal stuff which isn't exposed to the end-user.
- Increase minor version when fixing a bug

When a library starts having blurred versioning it makes things confusing, for example, Fable.Core had some problem like that in the past and in the end, everyone was confused (maintainers included^^).

**Comment by alfonsogarciacaro** [comment]
Thanks @MangelMaxime. We followed a similar strategy with Fable.Browser.XXX packages. To avoid pollution types are in the `Browser.Types` namespace while helpers for all packages are in `Browser`. Because namespaces cannot have values, we use `AutoOpen` modules ([example](https://github.com/fable-compiler/fable-browser/blob/4aa055287799ba96ce019a8674f53296a263a3a2/src/Event/Browser.Event.fs#L84-L93)) so `open Browser` makes all the helpers from referenced packages available. Of course you need to be careful there are no naming conflicts between packages. In the case of Fable.Browser it's ok because the mimic browser native APIs that are globally available and don't conflict with each other.

**Comment by Dzoukr** [comment]
Aaaaaaaaaaaaand done. All components just released with `v2.0.0` version so no need to open specific namespace anymore. Thanks for hint guys!
